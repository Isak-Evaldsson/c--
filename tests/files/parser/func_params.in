int test_param_1() {}
int test_param_2(int a) {}
int test_param_3(int a, int b, int c) {}