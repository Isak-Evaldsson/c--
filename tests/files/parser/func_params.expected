Prototype: test_param_1
Prototype: test_param_2
    Param: a
Prototype: test_param_3
    Param: a
    Param: b
    Param: c
